controllers/emailService.js

```diff
@@ -0,0 +1,39 @@
+import sgMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+
+const Url = process.env.NODE_ENV === 'development'
+  ? `http://localhost:${process.env.PORT}/api/v1/activate/`
+  : process.env.HEROKU_URL;
+
+const subject = 'Verification Email from LearnGround';
+const from = 'noreply@learncode.academy';
+
+export default {
+  sendWelcomeMail(email, userId, next) {
```

Comment:
Also, please find the comment I made initially and check if you missed anything.

Hint: This function is still not reusable. It should not know anything about the mail body.

It can take email, userId, body, subject that can be passed to it wherever it needs to be used.